sygnal/apnspushkin.py

```diff
@@ -230,7 +230,7 @@ async def _dispatch_request(
 
         code = int(response.status)
 
-        span.set_tag(tags.HTTP_STATUS_CODE, code)
+        span.set_tag(str(tags.HTTP_STATUS_CODE), code)
```

Comment:
I'm not sure why but on my install `HTTP_STATUS_CODE` is a string:
```
>>> import opentracing.tags
>>> opentracing.tags.HTTP_STATUS_CODE
'http.status_code'
```
I'm on opentracing 2.4.0 for reference

Reply:
> I'm not sure why but on my install HTTP_STATUS_CODE is a string

With context and thought, this sounds perfectly reasonable to me — it's a standardised name (key) for the tag where you put the status code of an HTTP request.

Reply:
I wondered about this but `opentracing/tags.py` definitely stated it was an int so I went with that: https://github.com/opentracing/opentracing-python/blob/3e1d357a348269ef54d67f761302fab93dbfc0f7/opentracing/tags.py#L108

It seems this is incorrect, I will change the type to match the implementation and not the comment.

Reply:
I'm guessing (haven't looked) that
- a span can be associated with multiple (tag, value) pairs.
- tag is a string identifying the meaning of the value. in this case, the string `'http.status_code'`
- value is probably an int!

Reply:
Ahhh that makes a lot of sense!

Reply:
I can see how the comment was a little misleading, but David's interpretation is what I believe also :)